includes/admin/settings/views/html-webhooks-edit.php

```diff
@@ -86,7 +86,7 @@
 
 							foreach ( $topics as $topic_slug => $topic_name ) :
 								?>
-							<option value="<?php echo esc_attr( $topic_slug ); ?>" <?php selected( $topic_data['topic'], $topic_slug, true ); ?>><?php echo esc_html( $topic_name ); ?></option>
+							<option value="<?php echo esc_attr( $topic_slug ); ?>" <?php selected( $topic_data['resource'] . '.' . $topic_data['event'], $topic_slug, true ); ?>><?php echo esc_html( $topic_name ); ?></option>
```

Comment:
This breaks the selecting of the 'Action' topic.

Reply:
As Claudiu said, this causes a problem.

Reply:
After a little more inspection, the issue is caused [by this](https://github.com/woocommerce/woocommerce/blob/094fc1bf3a25bd5e42becabbb219a7d9f52077ea/includes/admin/class-wc-admin-webhooks.php#L348-L349). For `subscription` the topic becomes `custom` which won't match any of the dropdown options.

I think @unfulvio's `selected` check for both should be a good solution. 👍

Reply:
thanks I've updated PR